Worker/Program.cs

```diff
@@ -10,29 +10,31 @@
 WriteLine("Witaj w edytorze twojej Listy Zakupów");
 if (back == false)
 {
-    WriteLine("Podaj imie i nazwisko Właściciela Listy Zakupów , jak i nazwę listy Zakupów");
-    ExampleList myList = new(ReadLine(), ReadLine(), ReadLine(),DateTime.Now);
-    WriteLine($"Utworzono Listę {myList.FirstName} {myList.LastName} o nazwie {myList.NameOfList} dnia {myList.LastDate}");
- 
+        WriteLine("Podaj imie i nazwisko Właściciela Listy Zakupów , jak i nazwę listy Zakupów");
+        ExampleList myList = new(ReadLine(), ReadLine(), ReadLine(),DateTime.Now);
+        WriteLine($"Utworzono Listę {myList.FirstName} {myList.LastName} o nazwie {myList.NameOfList} dnia {myList.LastDate}");
+
         if (start == true)
         {
             WriteLine($"Podaj nazwę, ilość i cenę pierwszego artykułu");
             Articles articles1 = new Articles()
             {
-                NameOfThing = ReadLine(),
-                amount = int.Parse(ReadLine()),
-                price = int.Parse(ReadLine()),
+                //NameOfThing = ReadLine(),
+                //amount = int.Parse(ReadLine()),
+                //price = decimal.Parse(ReadLine()),
             };
+            articles1.CreateANewArticle();    
```

Comment:
articles parameters should be read here and pass to a function (or constructor) of articles. Articles class should not be dependend on source (in this case - console)